src/components/navbar/NavBar.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { LinkButton } from '../../components';
+import { Menu, Dropdown, Typography, Row, Col, Button } from 'antd';
+import { DownOutlined, UserOutlined } from '@ant-design/icons';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+const { Text } = Typography;
+
+const NavRow = styled(Row)`
+  background-color: white;
+  padding: 1em 0;
+`;
+const AntButtonLink = styled(LinkButton)`
+  color: black;
+  margin-left: 7em;
+`;
+const Logo = styled.img`
+  float: right;
+`;
+const LogoText = styled(Text)`
+  font-size: 24px;
+`;
+const UserDropdown = styled(Button)`
+  float: right;
+  margin-right: 3em;
+`;
+
+const LoginSignupButtons = styled.div`
+  float: right;
+  margin-right: 3em;
+`;
+
+const SignupButton = styled(Button)`
+  margin-left: 1em;
+`;
+
+const NavBar: React.FC = () => {
+  const [authenticated] = useState(false);
+
+  const userMenu = (
+    <Menu>
+      <Menu.Item>Change Primary Account Email</Menu.Item>
+      <Menu.Item>Account Details</Menu.Item>
+      <Menu.Item>Change Password</Menu.Item>
+      <Menu.Item>Deactivate Account</Menu.Item>
+    </Menu>
+  );
+
+  return (
+    <NavRow align="middle">
+      <Col flex={1}>
+        <Link to="/">
+          <Logo
```

Comment:
Let's remove the logo from the frontend-scaffold

Reply:
Ok, this will break a bunch of CSS though